polkadot-protocol/architecture/system-chains/overview.md

```diff
@@ -1,30 +1,30 @@
+Polkadot's relay chain is designed to secure rollups and facilitate seamless inter-chain communication. However, resource-intensive—tasks like governance, asset management, and bridging are more efficiently handled by system rollups. These specialized chains offload functionality from the relay chain, leveraging Polkadot's parallel execution model to improve performance and scalability. By distributing key functionalities across system rollups, Polkadot can maximize its relay chain's blockspace for its core purpose of securing and validating rollups.
 
-This guide will explore how system parachains operate within Polkadot and Kusama, detailing their critical roles in network governance, asset management, and bridging. You'll learn about the currently deployed system parachains, their unique functions, and how they enhance Polkadot's decentralized ecosystem.
+This guide will explore how system rollups operate within Polkadot and Kusama, detailing their critical roles in network governance, asset management, and bridging. You'll learn about the currently deployed system rollups, their unique functions, and how they enhance Polkadot's decentralized ecosystem.
 
 ## System Chains
 
-System parachains contain core Polkadot protocol features, but in parachains rather than the relay chain. Execution cores for system chains are allocated via network [governance](/polkadot-protocol/onchain-governance/overview/){target=\_blank} rather than purchasing coretime on a marketplace.
+System rollups contain core Polkadot protocol features, but in rollups rather than the relay chain. Execution cores for system chains are allocated via network [governance](/polkadot-protocol/onchain-governance/overview/){target=\_blank} rather than purchasing coretime on a marketplace.
 
-System parachains defer to on-chain governance to manage their upgrades and other sensitive actions as they do not have native tokens or governance systems separate from DOT or KSM. It is not uncommon to see a system parachain implemented specifically to manage network governance.
+System rollups defer to on-chain governance to manage their upgrades and other sensitive actions as they do not have native tokens or governance systems separate from DOT or KSM. It is not uncommon to see a system rollups implemented specifically to manage network governance.
 
 !!!note
-    You may see system parachains called common good parachains in articles and discussions. This nomenclature caused confusion as the network evolved, so system parachains is preferred. 
+    You may see system rollups called common good rollups in articles and discussions. This nomenclature caused confusion as the network evolved, so system rollups is preferred. 
```

Comment:
```suggestion
    You may see system rollups called common good rollups (or parachains) in articles and discussions. This nomenclature caused confusion as the network evolved, so system rollups is preferred. 
```

Maybe here we should also say parachains since we're explicitly calling out these discrepancies

Reply:
This is a weird one, because no one said `common good rollups` anywhere. The issue was between system parachains and common good parachains. Not sure how to fix this

Reply:
Ok, what about something like this then:

> You may see system rollups referred to as system parachains or common good parachains in older articles and discussions. As the network evolved, this terminology became outdated, and system rollups is now preferred.